pennylane/allocation.py

```diff
@@ -154,14 +154,14 @@ def deallocate(wires: DynamicWire | Wires | Sequence[DynamicWire]) -> Deallocate
         def c():
             qml.H(0)
 
-            wires = qml.allocation.allocate(1, require_zeros=True)
+            wires = qml.allocate(1, require_zeros=True)
```

Comment:
```suggestion
            wires = allocate(1, require_zeros=True)
```
How about just without the imports for the moment?

Reply:
Even though we are showing these publicly, I still kinda want the docstrings to be accurate.

Reply:
the example below should work come release time, right? (i.e., just a circuit drawing—side note, we should just make this a qfunc, not a QNode 😬)

In any case, I think the warning at the top saying that this is experimental suffices :). If a user really wanted to tinker with this feature for some reason, no reason why we should make their lives harder by being mischievous with the path 😅